build.sbt

```diff
@@ -142,6 +142,17 @@ lazy val http4s = project
   )
   .dependsOn(coreJVM)
 
+lazy val play_base = project
+  .in(file("play"))
+  .settings(name := "caliban-play", commonSettings)
```

Comment:
Isn’t it going to try to publish the same binary twice?

Reply:
No this should just do it once because only the specific builds are being added to aggregate (is my understanding at least)